client/src/test/runTest.ts

```diff
@@ -22,23 +22,32 @@ import * as path from 'path';
 
 import { runTests } from 'vscode-test';
 
-async function main() {
+async function main(): Promise<number> {
 	try {
 		// The folder containing the Extension Manifest package.json
 		// Passed to `--extensionDevelopmentPath`
 		const extensionDevelopmentPath = path.resolve(__dirname, '../../');
 
 		// The path to the extension test script
 		// Passed to --extensionTestsPath
-		const extensionTestsPath = path.resolve(__dirname, './index');
+		const extensionTestsPath = path.resolve(__dirname, 'index.js');
 
+		const testOption = { 
+			extensionDevelopmentPath: extensionDevelopmentPath, 
+			extensionTestsPath: extensionTestsPath 
+		};
 		// Download VS Code, unzip it and run the integration test
-		await runTests({ extensionDevelopmentPath, extensionTestsPath });
+		const res = await runTests(testOption);
+		return Promise.resolve(res);
```

Comment:
```suggestion
		return runTests(testOption);
```

And once again, the function does not need to be `async`.